product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -30,7 +30,7 @@ EDB provides a shell script called [`biganimal-preflight-azure`](https://github.
 
   ```sh
    curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/biganimal-preflight-azure | bash -s \
```

Comment:
```suggestion
   curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/biganimal-preflight-azure | bash -s 
```

Reply:
@ftouserkani-edb I think we should remove the line break from this line, do you agree?

Reply:
@xugy99 , can you review and comment on whether or not we should remove the line break? I think we should.

Reply:
we can't remove it, as well as the `--` at L33, they are all used by the same single-line command.

Reply:
But, running the actual script, the script usage indicated the "--" is **NOT** required and I have verified the script **DOES** work without "--" in the command line.  

additionally, some of the **"Options"** names are inconsistent from what the script outputs currently, compared to what is in the UI, as the script was updated recently.

All I wanted/want to do is to provide consistencies across the board since this is a client facing documentation.

frank.touserkani@laptop615-ma-us azure % ./biganimal-preflight-azure

Usage: biganimal-preflight-azure [options] **<target-subscription>** <region>

Arguments:
    <target-subscription>    Azure subscription for BigAnimal deployment
    <region>                          Azure region for BigAnimal deployment

Options:
    -h, --help                     Print this help message
    -i, --instance-type      Azure VM instance for BigAnimal cluster, e.g., e2s_v3
    -a, --high-availability  Plan for BigAnimal cluster with high availability enabled
    -e, --endpoint              Network endpoint flavor for BigAnimal cluster
    -r, --activate-region    Include region activation, if no clusters exist in region
    --onboard                     Check if the user and subscription are configured appropriately

Behavior defaults to --onboard if no other options provided.

Examples:
    biganimal-preflight-azure --onboard 12412ab3d-1515-2217-96f5-0338184fcc04 eastus2
    biganimal-preflight-azure -i e2s_v3 --high-availability -e private 12412ab3d-1515-2217-96f5-0338184fcc04 eastus2

Available regions are: australiaeast brazilsouth canadacentral centralus eastus eastus2 francecentral japaneast northeurope norwayeast southcentralus southeastasia uksouth westeurope westus2 westus3
Available instance types are: e2s_v3 e4s_v3 e8s_v3 e16s_v3 e20s_v3 e32s_v3 e48s_v3 e64s_v3
Available endpoint flavors: public private

Reply:
Please take a look at the actual output as Confluence is not dealing with "< >" correctly

![preflight](https://user-images.githubusercontent.com/94471063/148972097-429a2c13-a8a0-426a-b441-f4971650cf1c.png)